modules/33acrossIdSystem.js

```diff
@@ -73,14 +73,41 @@ function calculateQueryStringParams(pid, gdprConsentData) {
     params.gdpr_consent = gdprConsentData.consentString;
   }
 
-  const fp = storage.getDataFromLocalStorage(STORAGE_FPID_KEY);
+  const fp = getStoredValue(STORAGE_FPID_KEY, storageConfig);
   if (fp) {
     params.fp = fp;
   }
 
   return params;
 }
 
+function handleFpId(fpId, storageConfig = {}) {
+  if (!fpId) {
+    return storage.removeDataFromLocalStorage(STORAGE_FPID_KEY);
+  }
+
+  storeValue(STORAGE_FPID_KEY, fpId, storageConfig);
+}
+
+function storeValue(key, value, storageConfig = {}) {
+  if (storageConfig.type === STORAGE_TYPE_COOKIES && storage.cookiesAreEnabled()) {
```

Comment:
What is `storage` is default of `{}` or `type` is not set? Also what is the expected structure of `storageConfig` the pub should pass? Have we documented this in the md?

Reply:
we won't be storing any values if the storage is `{}` or the type is not set.. that's the behavior that can be found in the baseID module `modules/userId/index.js` as well.

The recommended structure is part of the documentation --> https://docs.prebid.org/dev-docs/modules/userid-submodules/33across.html

```
pbjs.setConfig({
  userSync: {
    userIds: [{
      name: "33acrossId",
      params: {
        pid: "0010b00002GYU4eBAH" // Example ID
      },
      storage: {
        name: "33acrossId",
        type: "html5",
        expires: 90,
        refreshInSeconds: 8 * 3600
      }
    }]
  }
});
```